web_app/calendar/app.py

```diff
@@ -52,6 +55,74 @@ def tasks_creator():
     return redirect(url_for_redirect)
 
 
+@app.route(tasks_edit_route + '/', methods=['get'])
+@app.route(tasks_edit_route, methods=['get'])
+def tasks_edit(task_id):
+    task = get_task_by_id(task_id)
+    url_for_redirect = get_parameter()
+
+    if task.get('calendar_date') is not None:
+        obj_date = datetime.datetime.strptime(
+            task.get('calendar_date'), "%Y-%m-%d %H:%M:%S")
+    else:
+        obj_date = datetime.datetime.now()
+
+    form = task_preset_form.format(
+        year=obj_date.year,
+        month=obj_date.month,
+        day=obj_date.day,
+        hour=obj_date.hour,
+        redirect=url_for_redirect)
+
+    # make calendar date hidden
+    if task.get('calendar_date') is None:
+        form = form.replace('checked="checked"', '')
+
+    # set name of task
+    form = form.replace('</textarea>', task.get('name') + '</textarea>')
+    # set id of task
+    form = form.replace('[task_id]', str(task_id))
```

Comment:
Why don't you use format instead of replace?

Reply:
format changes '{}' and it makes us add '{' in HTML